app/policies/chapter_policy.rb

```diff
@@ -9,6 +9,20 @@ def create?
   end
 
   def show?
-    user.is_admin? or user.has_role?(:organiser, record) or user.has_role?(:organiser)
+    is_admin_or_organiser?
+  end
+
+  def edit?
+  	is_admin_or_organiser?
```

Comment:
@notthepoint Would be nice to have the two space indentation here and similarly in other places. What do you think ?